### PR TITLE
[Editor] Update some strings (bug 1787299)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: l10n/en-US/viewer.properties [hunk 1]
```diff
@@ -251,21 +251,21 @@ printing_not_ready=Warning: The PDF is not fully loaded for printing.
...
-editor_free_text_color=Color
-editor_free_text_size=Size
-editor_ink_color=Color
-editor_ink_thickness=Thickness
-editor_ink_opacity=Opacity
+editor_text_color=Color
+editor_text_size=Size
+editor_draw_color=Color
+editor_draw_thickness=Thickness
+editor_draw_opacity=Opacity
```
Comment (Snuffleupagus) [inline l10n/en-US/viewer.properties:266]: Here, and elsewhere, why are we changing the string-ids when the content doesn't actually change?
Won't this cause localizers to (unnecessarily) have to re-translate this?

Can we please just revert these changes?

File: l10n/en-US/viewer.properties [hunk 2]
```diff
@@ -251,21 +251,21 @@ printing_not_ready=Warning: The PDF is not fully loaded for printing.
...
 
 # Editor
-editor_free_text.title=Add FreeText Annotation
-editor_free_text_label=FreeText Annotation
-editor_ink.title=Add Ink Annotation
-editor_ink_label=Ink Annotation
+editor_text.title=Text
+editor_text_label=Text
+editor_draw.title=Draw
+editor_draw_label=Draw
```
Comment (Snuffleupagus) [inline l10n/en-US/viewer.properties:257]: Here, and elsewhere, I'm not crazy about the new string-ids. Basically it seems preferable to me if the string-ids themselves are still more inline with the actual code, since without prior knowledge it'd not be immediately obvious that e.g. "editor_draw" refers to the Ink-annotation.

It's perhaps a bit less nice, but how about something like this (a pattern we already use elsewhere)?
```suggestion
editor_free_text2.title=Text
editor_free_text2_label=Text
editor_ink2.title=Draw
editor_ink2_label=Draw
```

File: l10n/en-US/viewer.properties (hunk 1 above)
Comment (calixteman) [reply l10n/en-US/viewer.properties:266]: It was just to be consistent, i.e. avoid having some free_text and text.
I thought that adding a number was a bit ugly but if you're fine with that then I am too.

**Review by Snuffleupagus — APPROVED** [review]
